apps/search/__init__.py

```diff
@@ -249,12 +250,13 @@ async def get_async(self, req, lookup) -> ElasticAsyncEveCursor:
 
         cursor = await self.elastic_async.search(query, types, params)
 
-        app = get_current_app().as_any()
-        for resource in types:
-            response = {ITEMS: [doc async for doc in cursor if doc["_type"] == resource]}
-            await getattr(app, "on_fetched_resource").call_async(resource, response)
-            await getattr(app, "on_fetched_resource_%s_async" % resource).call_async(response)
-            await getattr(app, "on_fetched_resource_%s" % resource).call_async(response)
+        if not req or not req.args or req.args.get("run_signals", True):
```

Comment:
I wonder if we should make it rather a kwarg in `get_async` because often we would be using it with
`..get_async(req=None)`, so `.get_async(req=None, signals=False)` would be easier than constructing request for that